src/parser/priest/discipline/modules/spells/Schism.tsx

```diff
@@ -88,6 +97,7 @@ class Schism extends Analyzer {
     this._lastSchismCast = event;
 
     // Assume every schism is bad
+    // @ts-ignore
```

Comment:
This probably needs to be cleaned up; right now the event itself is being used to index the object, could that be changed to index on the events timestamp?